### ISSUE TITLE
Unknown ctr table merge policy FailIfCtrsIntersects

### ISSUE THREAD

**Issue by ibuda** [catboost/catboost#2497]
Problem: Unknown ctr table merge policy FailIfCtrsIntersects
catboost version: 1.2.2
Operating System: Ventura Version 13.5.2 (22G91)
CPU: Apple M1 Pro
GPU:

When I am trying to sum models using:
```
model = sum_models(
    [model_low, model_high], 
    weights=[0.5, 0.5],
    ctr_merge_policy='FailIfCtrsIntersects'
)
```
I get the following error:
```
----> 1 model = sum_models(
      2     [model_low, model_high], 
      3     weights=[0.5, 0.5],
      4     ctr_merge_policy='FailIfCtrsIntersects'
      5 )

File ~/Library/Python/3.9/lib/python/site-packages/catboost/core.py:6821, in sum_models(models, weights, ctr_merge_policy)
   6819 def sum_models(models, weights=None, ctr_merge_policy='IntersectingCountersAverage'):
   6820     result = CatBoost()
-> 6821     result._sum_models(models, weights, ctr_merge_policy)
   6822     return result

File ~/Library/Python/3.9/lib/python/site-packages/catboost/core.py:1843, in _CatBoostBase._sum_models(self, models_base, weights, ctr_merge_policy)
   1841     weights = [1.0 for _ in models_base]
   1842 models_inner = [model._object for model in models_base]
-> 1843 self._object._sum_models(models_inner, weights, ctr_merge_policy)
   1844 setattr(self, '_random_seed', 0)
   1845 setattr(self, '_learning_rate', 0)

File _catboost.pyx:5103, in _catboost._CatBoost._sum_models()

File _catboost.pyx:5109, in _catboost._CatBoost._sum_models()

CatBoostError: Unknown ctr table merge policy FailIfCtrsIntersects
```

**Comment by lmacken** [comment]
Try changing the `ctr_merge_policy` to `FailIfCtrIntersects`.

**Comment by andrey-khropov** [comment]
Fixed in #2499
